### ISSUE TITLE
Porting for a pure C implementation

### ISSUE THREAD

**Issue by Dinesh-Ramakrishnan** [emilk/loguru#236]
Hi @emilk , 

Thank you for this wonderful and super useful library.
I have a pure C application in which I would like to use this logging library. Do we have any C language port available for this library?

If not, please provide me some guidelines, I am interested to port this to pure C library.

**Comment by BullyWiiPlaza** [comment]
I don't think so, man. This is a C++ library. You cannot simply provide a C implementation without rewriting it almost entirely. C++ streams won't be possible and so on. You're better off finding another logging library specifically for C.

**Comment by virtuosonic** [comment]
Rewritting any mature software will always result in an inmature and maybe buggy software. Making a wrapper would be complicated because of macros usage, some features depend on C++ RAII. Have you tried compiling your C application with a C++ compiler, maybe that is less work
